tests/integration/standard/test_authentication.py

```diff
@@ -42,12 +43,19 @@ def setup_module():
         log.debug("Starting ccm test cluster with %s", config_options)
         start_cluster_wait_for_up(ccm_cluster)
 
+    # PYTHON-1328
+    #
+    # Give the cluster enough time to startup (and perform necessary initialization)
+    # before executing the test.
+    if CASSANDRA_VERSION > Version('4.0-a'):
```

Comment:
Version string here pulled from the [annotations](https://github.com/datastax/python-driver/blob/master/tests/integration/__init__.py#L333-L335) defined in `tests/integration/__init__.py`